report/chap/parity.tex

```diff
@@ -1,95 +0,0 @@
-\chapter{Parity}
-
-Parity games were introduced in \cite{emersonjutla} to solve the $\mu$-calculus model checking problem. From the view of Complexity Theory, the problem of deciding the winner of a parity game is in $\NP \cap \coNP$ (so unlikely to be $\NP$-complete), but the question of whether it has a polynomial time solution remains open.
```

Comment:
Add \usepackage{hyperref}. In the pdf, when we move the pointer close to the citation, it displays the reference as an overlay. This is quite convenient while reading.